source/app/states/Maksym Degtyar/max.html

```diff
@@ -0,0 +1,27 @@
+<body>
```

Comment:
body already defined in index.html. Using body tag within body is mistake. Please fix.